### ISSUE TITLE
Errors with gif resizing to larger frame sizes

### ISSUE THREAD

**Issue by mike-hart** [imgproxy/imgproxy#851]
Running the latest imgproxy via `docker run -e IMGPROXY_LOG_LEVEL=debug -p 8081:8080 -it darthsim/imgproxy:v3.4.0` along with the image attached, and accessing the image provided (though a similar outcome is likely for any gif I've seen): `/insecure/resize:fit:2400:1800:0/gravity:ce/plain/<url_to_source_image>` results in:

```
error="gifsave_buffer: frame too large\n"
/app/vips/vips.go:282 github.com/imgproxy/imgproxy/v3/vips.(*Image).Save
/app/processing/processing.go:270 github.com/imgproxy/imgproxy/v3/processing.ProcessImage
/app/processing_handler.go:305 main.handleProcessing.func3
/app/processing_handler.go:303 main.handleProcessing
/app/server.go:103 main.withCORS.func1
/app/server.go:150 main.withPanicHandler.func1
/app/router/router.go:102 github.com/imgproxy/imgproxy/v3/router.(*Router).ServeHTTP
/usr/local/go/src/net/http/server.go:2880 net/http.serverHandler.ServeHTTP
/usr/local/go/src/net/http/server.go:1931 net/http.(*conn).serve
/usr/local/go/src/runtime/asm_amd64.s:1582 runtime.goexit
```

Using a smaller frame (say, 800x600) does not suffer the same outcome. This did not occur in v2.17.0, and seems to have first started in v3.0.0.

![example](https://user-images.githubusercontent.com/4129199/164505009-ca1b771a-785a-4d0e-a8e3-170a5e06a544.gif)

**Comment by mike-hart** [comment]
"latest" (Digest: sha256:9585099e8f87f8bcefdeb86637707c203c907d2e439c0123b4bf9ff72096871c) has a little more detail in the `error` field:
```
vips_image_get: field "iptc-data" not found
vips_image_get: field "xmp-data" not found
gifsave_buffer: frame too large
```

**Comment by DarthSim** [comment]
Hi @mike-hart!

`vips_image_get` errors are not related to this issue. GIF frame size is limited by libvips, and we're working on fixing this. Previously GIFs were saved with ImageMagick, but we removed IM from imgproxy v3.

**Comment by mike-hart** [comment]
Thanks @DarthSim - I mostly wanted to ensure you had sufficient information in the report. I hope it all helps.

**Comment by DarthSim** [comment]
Hey @mike-hart!
Finally, this issue is fixed in vips 8.13 and imgproxy 3.7.0
